src/taxonomy/TaxonomyListPage.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import {
+  Button,
+  CardView,
+  Container,
+  DataTable,
+  FormControl,
+  Spinner,
+} from '@edx/paragon';
+import { Add } from '@edx/paragon/icons';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import Header from '../header';
+import SubHeader from '../generic/sub-header/SubHeader';
+import messages from './messages';
+import TaxonomyCard from './TaxonomyCard';
+import { useTaxonomyListDataResponse, useIsTaxonomyListDataLoaded } from './api/hooks/selectors.ts';
+
+const TaxonomyListPage = ({ intl }) => {
+  const orgDefaultValue = intl.formatMessage(messages.orgInputSelectDefaultValue);
+
+  const useTaxonomyListData = () => {
+    const taxonomyListData = useTaxonomyListDataResponse();
+    const isLoaded = useIsTaxonomyListDataLoaded();
+    return { taxonomyListData, isLoaded };
+  };
+
+  const { taxonomyListData, isLoaded } = useTaxonomyListData();
+
+  const getHeaderButtons = () => [
+    (
+      <Button
+        variant="link"
+        className="text-dark-900"
+      >
+        {intl.formatMessage(messages.downloadTemplateButtonLabel)}
+      </Button>
+    ),
+    (
+      <Button
+        iconBefore={Add}
+      >
+        {intl.formatMessage(messages.importButtonLabel)}
```

Comment:
The import button is out of scope for now (comes later in #115), and might be confusing if it's present but doesn't work. If you do leave it in, please make sure it's disabled and maybe add a tooltip that says "Import feature coming soon".